cloud/scope/powervs_machine.go

```diff
@@ -397,30 +397,28 @@ func (m *PowerVSMachineScope) Name() string {
 
 func (m *PowerVSMachineScope) createIgnitionData(data []byte) (string, error) {
 	if len(data) == 0 {
-		return "", fmt.Errorf("got empty data")
+		return "", fmt.Errorf("user data is empty")
 	}
 
 	cosClient, err := m.createCOSClient()
 	if err != nil {
-		m.Error(err, "failed to create cosClient")
-		return "", fmt.Errorf("failed to create cosClient %w", err)
+		return "", fmt.Errorf("failed to create COS client %w", err)
 	}
 	key := m.bootstrapDataKey()
-	m.Info("bootstrap data key", "key", key)
+	m.V(3).Info("Bootstrap data key details", "key", key)
```

Comment:
Its just a name, so may be
```suggestion
	m.V(3).Info("Bootstrap data key name", "key", key)
```